jwt.test-d.ts

```diff
@@ -26,14 +26,14 @@ const secretOptions = {
 const jwtOptions: FastifyJWTOptions = {
   secret: 'supersecret',
   sign: {
-    expiresIn: '1h'
+    expiresIn: 36000
   },
   cookie: {
     cookieName: 'jwt',
     signed: false
   },
   verify: {
-    maxAge: '1h',
+    maxAge: 36000,

```

Comment:
shall we not have tests also with strings?

Reply:
We do have them here: 
- https://github.com/fastify/fastify-jwt/blob/3665579076b65f0cdcefe6fd0f797524fdd2e740/test/test.test.js#L2478
- https://github.com/fastify/fastify-jwt/blob/3665579076b65f0cdcefe6fd0f797524fdd2e740/test/test.test.js#L2499

Reply:
these are typings tests, they're two different things :)

Reply:
They were not covered in the types and triggered typescript errors, I have added some suggestions in order to make strings describing a time span valid.
However it can be improved by adding a JSDoc example covering string time span use case to make it easier for the end user.